frontends/web/src/routes/lightning/send/send.tsx

```diff
@@ -64,13 +63,15 @@ export function Send({ accounts, code }: Props) {
     }
   };
 
-  const onRawInputChange = (event: ChangeEvent<HTMLInputElement>) => {
-    const target = event.target as HTMLInputElement;
-    setRawInput(target.value);
-  };
-
-  const onChangeActiveScanQr = (active: boolean) => {
-    setActiveScanQr(active);
+  const onInvoice = (invoice: string) => {
+    // TODO: ensure this is a valid invoide?
+    if (!invoice) {
+      setRawInputError('is not a valid invoice');
+      return;
+    }
+    setRawInputError(undefined);
```

Comment:
I would suggest moving this line into the `parseInput` function to stop the flickering, maybe around L83

Reply:
Moved, but it still flickers quickly. Looks like it is coming from:

```
err Error: SdkError: Generic: Breez SDK error: Unrecognized input type
    at getApiResponse (lightning.ts:695:11)
    at async send.tsx:75:22
```

The reason was the the rawInput was first stored in a state, and then later (after the state update was done) it calls parseInput. I removed storing the invoice in a state and call parseInput directly with the invoice now.